### ISSUE TITLE
On unRAID, the file permissions for the docker container are incorrect due to the umask.

### ISSUE THREAD

**Issue by DavidSpivey** [linuxserver/docker-syncthing#16]
When using SyncThing on unRAID, the permissions set by the umask in your docker container change the permissions to deny write permissions to users over Samba file sharing. I would love to use this docker container on unRAID, but when using it, I cannot modify the files synced to my server.

The current umask is 022, which removes permissions. The correct umask is 000, which permits write access.

In other words, the change made on 30.09.16 is incorrect as well.

**Comment by aptalca** [comment]
Perhaps 002 would be more appropriate, as it would give write permission to group members but not everyone else.

On unraid, all smb users as well as the default docker user belong to group `nobody`

**Comment by DavidSpivey** [comment]
While that may seem to be the correct course of action, unRAID itself and the "New Permissions" tool in unRAID both set 666 permissions on files. You are suggesting 664. I personally don't care so long as I can access the files with no issues, but why do something different than unRAID itself does?

**Comment by sparklyballs** [comment]
most linux distros set 022 as default umask so the change was not incorrect

our user base is wider than just unraid

however the best course of action is to allow setting of umask via an env in the run command (via dockerman for unraid)

there will be a PR for this very shortly (as in i'm typing it up now)

**Comment by DavidSpivey** [comment]
I apologize. I was not aware that this docker container was in use by others. Thank you for your help. I know that many of the variables for the docker container on unRAID are automatically set when creating the container such as PUID and PGID. Will this umask also be 000 by default when creating the container on unRAID?
